From f51473b281bca2d101ee5ac0ff565d939c053b98 Mon Sep 17 00:00:00 2001
From: Karthik Kambatla <kasha@apache.org>
Date: Mon, 3 Oct 2016 06:03:46 -0700
Subject: [PATCH 2065/2748] YARN-5672. FairScheduler: Wrong queue name in log
 when adding application. (Wilfred Spiegelenburg
 via kasha)

(cherry picked from commit 0da54e8848764c71a31473516d23ada582013f8c)

Change-Id: I55982b9ad50d4f4cf94b5e298c67f0e62442cde0
---
 .../scheduler/fair/FairScheduler.java              |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FairScheduler.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FairScheduler.java
index 064bffb..dcb8d2f 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FairScheduler.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FairScheduler.java
@@ -663,11 +663,12 @@ protected synchronized void addApplication(ApplicationId applicationId,
     queue.getMetrics().submitApp(user);
 
     LOG.info("Accepted application " + applicationId + " from user: " + user
-        + ", in queue: " + queueName + ", currently num of applications: "
-        + applications.size());
+        + ", in queue: " + queue.getName()
+        + ", currently num of applications: " + applications.size());
     if (isAppRecovering) {
       if (LOG.isDebugEnabled()) {
-        LOG.debug(applicationId + " is recovering. Skip notifying APP_ACCEPTED");
+        LOG.debug(applicationId
+            + " is recovering. Skip notifying APP_ACCEPTED");
       }
     } else {
       rmContext.getDispatcher().getEventHandler()
-- 
1.7.9.5

